src/discussions/common/AuthorLabel.test.jsx

```diff
@@ -61,7 +61,7 @@ describe('Author label', () => {
 
   describe.each([
     ['anonymous', null, false, ''],
-    ['ta_user', 'Community TA', true, 'text-TA-color'],
+    ['ta_user', 'Moderator', true, 'text-TA-color'],
```

Comment:
I think you should keep the original test case options as it is like 
['ta_user', 'Moderator', true, 'text-TA-color'] and add another test case for 'Moderator'